client/src/classes/PeerProcessor.ts

```diff
@@ -12,28 +12,34 @@ export class PeerProcessor {
   peerConnection: RTCPeerConnection;
   dataChannel: RTCDataChannel;
   socket: Socket;
-  sender: RTCRtpSender;
+  audioSender: RTCRtpSender;
+  videoSender: RTCRtpSender;
```

Comment:
Not sure if we will ever implement video from camera, but if so, can the new video stream be added to this same `videoSender`?

Reply:
I think if there's a new video stream, I should be able to just take the track and add it to the current video stream?
I'm actually not even sure if I need separate logic for video and audio...??? hmmmm